File: app/src/main/java/com/agera/hometools/network/Callback.java
package com.agera.hometools.network;


import com.google.android.agera.net.HttpResponse;

/**
 * Created by  Agera K on 2017/9/22.
 */
public abstract class Callback {
    public abstract void success(HttpResponse result);

    public void error(Throwable e) {

    }

}

File: app/src/main/java/com/agera/hometools/network/Callback.kt
package com.agera.hometools.network

import com.google.android.agera.net.HttpResponse

/**
 * Created by Agera on 2017/11/8.
 */
abstract class Callback {

    abstract fun success(resp: HttpResponse)

    fun error(e: Throwable) {
    }
}